src/components/shared/Accordion.css

```diff
@@ -59,7 +59,3 @@
   width: 16px;
   height: 16px;
 }
-
-.accordion .header-buttons button::-moz-focus-inner {
```

Comment:
I'm not sure if this should be deleted. `autoprefixer` cannot add this. Any ideas?